components/nds-page-header/src/PageHeader.tsx

```diff
@@ -1,54 +1,111 @@
 import React from "react";
+import { Grid, GridItem } from "@nice-digital/nds-grid";
 
 import "../scss/page-header.scss";
 
 export interface PageHeaderProps {
 	[prop: string]: unknown;
 	useAltHeading?: boolean;
 	preheading?: React.ReactNode;
+	breadcrumbs?: React.ReactNode;
 	heading: React.ReactNode;
-	lead?: React.ReactNode;
 	metadata?: React.ReactNode[];
+	lead?: React.ReactNode;
+	description?: React.ReactNode;
 	cta?: React.ReactNode;
+	secondSection?: React.ReactNode;
+	variant?: "normal" | "fullWidthDark" | "fullWidthLight";
 }
 
 export const PageHeader: React.FC<PageHeaderProps> = (
 	props: PageHeaderProps
 ) => {
 	const {
-		heading,
 		useAltHeading = false,
-		lead,
+		preheading,
+		breadcrumbs,
+		heading,
 		metadata,
+		lead,
+		description,
 		cta,
-		preheading,
+		secondSection,
+		variant = "normal",
 		...rest
 	} = props;
 
-	return (
-		<div className="page-header" data-component="page-header" {...rest}>
-			<h1
-				className={`page-header__heading ${
-					useAltHeading ? "page-header__heading--alt" : ""
-				}`}
-			>
-				{preheading && (
-					<span className="page-header__pre-heading">{preheading}</span>
+	// Figure out variant
+	const isFullWidth =
+		variant === "fullWidthDark" || variant === "fullWidthLight";
+	let variantClassname = "";
+	switch (variant) {
+		case "fullWidthDark":
+			variantClassname = "page-header--full-width-dark";
+			break;
+		case "fullWidthLight":
+			variantClassname = "page-header--full-width-light";
+			break;
+	}
+
```

Comment:
This is a possible alternative to the switch, not sure if it's more readable/maintainable however, especially if more variants get added so probably stick with switch for now...

`// Figure out variant
const isFullWidth =
  variant === "fullWidthDark" || variant === "fullWidthLight";
const variantClassname =
  isFullWidth && variant
    ? `page-header--${variant === "fullWidthDark" ? "full-width-dark" : "full-width-light"}`
    : "";`

Reply:
Ta John

Might stick with the switch for readability, but that deffo works with less code - nice one